hack/deploy-apiserver.sh

```diff
@@ -4,7 +4,7 @@ export SVC_NAME="${SVC_NAME:-hiveapi}"
 export SVC_NAMESPACE="${SVC_NAMESPACE:-hive}"
 
 oc process --local=true -f "config/apiserver/apiserver.template.yaml" \
-  -p CA_CRT="$(base64 -w0 config/apiserver/certificates/apiserver_ca.crt)" \
-  -p TLS_CRT="$(base64 -w0 config/apiserver/certificates/apiserver.crt)" \
-  -p TLS_KEY="$(base64 -w0 config/apiserver/certificates/apiserver.key)" \
+  -p CA_CRT="$(base64 -w0 hiveapi-certs/apiserver_ca.crt)" \
```

Comment:
Is this file still used?

Reply:
once we retire the ./hack/make-apiserver-certs.sh and ./hack/deploy-apiserver.sh scripts, these should go away